lib/rpmds.c

```diff
@@ -1278,6 +1278,11 @@ static const struct rpmlibProvides_s rpmlibProvides[] = {
     { "rpmlib(RichDependencies)",    "4.12.0-1",
 	(		RPMSENSE_EQUAL),
     N_("support for rich dependencies.") },
+#if HAVE_ZSTD_H
+    { "rpmlib(PayloadIsZstd)",		"5.4.18-1",
```

Comment:
The version was dictated by where first implemented. See the equivalent in xzdio/lzdio tracking dependencies.